config/core/configmaps/sugar.yaml

```diff
@@ -0,0 +1,56 @@
+# Copyright 2022 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: config-sugar
+  namespace: knative-eventing
+  labels:
+    eventing.knative.dev/release: devel
+    app.kubernetes.io/version: devel
+    app.kubernetes.io/name: knative-eventing
+  annotations:
+    knative.dev/example-checksum: "2f817977"
+data:
+  _example: |
+    ################################
+    #                              #
+    #    EXAMPLE CONFIGURATION     #
+    #                              #
+    ################################
+    # This block is not actually functional configuration,
+    # but serves to illustrate the available configuration
+    # options and document them in a way that is accessible
+    # to users that `kubectl edit` this config map.
+    #
+    # These sample configuration options may be copied out of
+    # this example block and unindented to be in the data block
+    # to actually change the configuration.
+
+    # namespace-sugar-selector specifies a LabelSelector which
+    # determines which namespaces the Sugar Controller should operate upon
+    # Use an empty value to disable the feature (this is the default):
+    namespace-sugar-selector: ""
+
+    # Use an empty object to enable for all namespaces
+    namespace-sugar-selector: {}
+
+    # trigger-sugar-selector specifies a LabelSelector which
+    # determines which triggers the Sugar Controller should operate upon
+    # Use an empty value to disable the feature (this is the default):
+    trigger-sugar-selector: ""
+
+    # Use an empty object to enable for all triggers
+    trigger-sugar-selector: {}
```

Comment:
There is some valuable extra example for end users at the test config. Do you think it could be added here?

https://github.com/knative/eventing/blob/c75ff0ebc66c7f463d5590eb49f1f5345fc66727/test/config/configmaps/config-sugar.yaml#L34-L39

Reply:
I was planning on capturing this in the release notes + updating knative.dev. Intentionally didn't add it to the config-sugar.yaml as this is the legacy way of doing things and now folks can much more flexibility.